### ISSUE TITLE
We might need a different criterion for determining Market Conditions.

### ISSUE THREAD

**Issue by RichardLuo0506** [mehtadone/PTFeeder#45]
The bear market for the past 3 days got me reconsidering how we determine if a market is bullish or bearish.

Assume BTC as base currency.
Generally speaking, when BTC goes up, ALTs go down, and vice versa. This isn't always the case, but it's true more often than not, so PTF has been great to me.
The past few days, both BTC and ALTs have been dipping. Would we classify the general trend to be bearish?
The way PTF currently use "MaxTopCoinAverageChange" and "MinBaseCoinPriceChange" to determine trend is problematic in the following example:

BTC drops 4%, ALTs drop 2% - In this case, even though everyone is losing nominal value, but arn't ALTs positive relative to BTC?

If BTC grows 100%, and ALTs grow 50%, PTF would call this a superbull market, but wouldn't we all be down 50% in bags?

I think we should have another check that compares ALTs trend to BTC trend. I bring this up because I believe we all trade with the assumption that both BTC and ALTs are bullish long term, so we're trading the difference.

Thanks for the good work!

**Comment by cryptroid** [comment]
You might be on to something....

**Comment by mehtadone** [comment]
I agree!! So something like MaxBaseAltDifference. If the difference between the alt change and the base coin (assume BTC) is less than the max, then its that market condition?

**Comment by lutzleonhardt** [comment]
+1 Yeah this sound meaningful.

**Comment by RichardLuo0506** [comment]
`I agree!! So something like MaxBaseAltDifference. If the difference between the alt change and the base coin (assume BTC) is less than the max, then its that market condition?`

Yes, but I'm not sure how the math would work for the scenarios.

If BTC rises by 1%, ALTs rise by 2%. Should "MaxBaseAltDifference" be 1%, 50%, or 100%?

Brain's not working.

**Comment by RichardLuo0506** [comment]
Okay I please check my math, I'm terrible at it:

ALT change relative to BTC change = ((ALT%/100+1)/(BTC%/100+1)-1)100

so if ALT goes up by 50%, BTC goes up by 100% you get:
((50/100+1)/(100/100+1)-1)*100 = -25%

if ALT goes up by 50%, BTC goes down by 50% you get: 
((50/100+1)/(-50/100+1)-1)*100 = 200%

if ALT goes down by 40%, BTC goes down by 60% you get:
((-40/100+1)/(-60/100+1)-1)*100 = 50%

**Comment by Tank33au** [comment]
No trading book would tell you to use price change % as a trend indicator.
I really hope to see EMAs (whether looking at a slope of EMA) or EMA(FAST)>EMA(SLOW) to determine the trend.

**Comment by RichardLuo0506** [comment]
> No trading book would tell you to use price change % as a trend indicator.
I really hope to see EMAs (whether looking at a slope of EMA) or EMA(FAST)>EMA(SLOW) to determine the trend.

Incremental improvements, I hope to see this with EMA together in the future.

**Comment by Tomokrypto** [comment]
Fantastic point, and great to see everyone working to improve this awesome bot!
